CONTRIBUTING.md

```diff
@@ -0,0 +1,46 @@
+# Contributing to `universum`
+
+## :wave: Greetings Traveler!
+
+I'm really glad you're reading this, I really appreciate the effort you're
+taking. Thank you for your help in making this library awesome! :sparkles:
+
+### How to contribute
+
+#### Report bugs or feature request
+If you found any bugs or have any proposals how to make this project better
```

Comment:
If you have found any bugs or have proposals on how to make this project better,